iOS/UI/Swipeable Input/SwipeableInputAccessoryView.swift

```diff
@@ -151,9 +160,27 @@ class SwipeableInputAccessoryView: BaseView, UIGestureRecognizerDelegate, Active
             self.didPressAlertCancel()
         }
     }
+    
+    func updateHeight(with numberOfLines: Int) {
+        var new: CGFloat = SwipeableInputAccessoryView.minHeight
```

Comment:
To keep the text view centered, and the bubble with a consistent initial height, we use this to update the bubble size based on number of lines.